### PR TITLE
Update web app tutorial

### PR DESCRIPTION
**Pull request by timothymcmackin**
Updates to the web app tutorial that I did while going through it for the first time.

Things to review in particular:
- My statements about how dApps and this dApp in particular behave
- Info about the LB contract and how we call it
- Info about what's actually happening when the app calls the contract

See the preview for a running version: https://docs-staging-git-feat-update-web-app-tutorial-trili-tech.vercel.app/tutorials/build-your-first-app/

In the long term, I'd like to have a dApp that will work on testnet; this does not because the LB contract on ghostnet has no tzBTC. I'd also like to have a less-complicated contract that is more coding and less "copy these files from the working application."

Before deploying, we must set the https://github.com/trilitech/tutorial-applications/ repo to public.

### CODE REVIEW

**Comment by sashaaldrick** [comment]
@claudebarde could you please review this as it builds on your work

File: src/pages/tutorials/build-your-first-app/setting-up-app.md [hunk 1]
```diff
@@ -0,0 +1,660 @@
...
+   };
+
+   export const shortenHash = (hash: string): string =>
+     hash ? hash.slice(0, 5) + "..." + hash.slice(-5) : "";
+
+   export const calcDeadline = () =>
+   new Date(Date.now() + 3_600_000).toISOString();
+   ```
+
+1. Copy these files from the tutorial repository at <https://github.com/trilitech/tutorial-applications/tree/main/liquidity-baking-dapp>, making sure to put the files in the sale folder in your application.
```
Comment (beatalipska) [inline src/pages/tutorials/build-your-first-app/setting-up-app.md:649]: This repo is private? we could write a simple shell script to get the files created and populated with the right content from github repo (this example is not working as the repo is private, but still would save a lot of time: 
```# Create directories
mkdir -p src/lib

# Download and save content from GitHub
curl -o src/config.ts https://raw.githubusercontent.com/trilitech/tutorial-applications/main/liquidity-baking-dapp/src/config.ts
curl -o src/store.ts https://raw.githubusercontent.com/trilitech/tutorial-applications/main/liquidity-baking-dapp/src/store.ts
curl -o src/types.ts https://raw.githubusercontent.com/trilitech/tutorial-applications/main/liquidity-baking-dapp/src/types.ts
curl -o src/lib/Interface.svelte https://raw.githubusercontent.com/trilitech/tutorial-applications/main/liquidity-baking-dapp/src/lib/Interface.svelte
curl -o src/lib/Sidebar.svelte https://raw.githubusercontent.com/trilitech/tutorial-applications/main/liquidity-baking-dapp/src/lib/Sidebar.svelte
curl -o src/lib/Toast.svelte https://raw.githubusercontent.com/trilitech/tutorial-applications/main/liquidity-baking-dapp/src/lib/Toast.svelte
curl -o src/lib/UserStats.svelte https://raw.githubusercontent.com/trilitech/tutorial-applications/main/liquidity-baking-dapp/src/lib/UserStats.svelte

```

Comment (timothymcmackin) [reply src/pages/tutorials/build-your-first-app/setting-up-app.md:649]: I have made the repo public.

Comment (beatalipska) [reply src/pages/tutorials/build-your-first-app/setting-up-app.md:649]: nice! So the command works now properly and that will save some time, there's another place we can use this way of copying the content, I'll point it out in a separate comment.

Comment (timothymcmackin) [reply src/pages/tutorials/build-your-first-app/setting-up-app.md:649]: Not sure about the shell script. If the user is not experienced with them, copying that large script might be more trouble than downloading the files manually.

File: src/pages/tutorials/build-your-first-app/swapping-tokens.md [hunk 2]
```diff
@@ -1,37 +1,42 @@
...
-Here is a screenshot of the UI you are aiming for:
+The text field on the left is editable; the user can enter the amount of XTZ or tzBTC to exchange or click the "Max" button to set the field to the total amount of that token in the wallet.
+The field on the right is not editable; it shows the corresponding amount that they will get of the other token.
+The button in the middle with the 2 arrows switches the input between XTZ and tzBTC.
 
-![Swap UI](/images/build-your-first-app/swap-ui.png "Swap UI")
+## Handling user input
 
-There are 2 text inputs, the one on the left is editable and will let the user input the amount of XTZ or tzBTC they want to exchange and the one on the right will be disabled and will display the corresponding amount they'll get in the other token. The button in the middle with the 2 arrows will allow the user to switch the input between XTZ and tzBTC.
+To set up user input in your application, copy these files from the tutorial repository at <https://github.com/trilitech/tutorial-applications/tree/main/liquidity-baking-dapp>:
```
Comment (beatalipska) [inline src/pages/tutorials/build-your-first-app/swapping-tokens.md:23]: Command for this: 
```
# Create directories if they don't exist
mkdir -p src/lib

# Download and save content from GitHub for UserInput.svelte
curl -o src/lib/UserInput.svelte "https://raw.githubusercontent.com/trilitech/tutorial-applications/main/liquidity-baking-dapp/src/lib/UserInput.svelte"

# Download and save content from GitHub for SwapView.svelte
curl -o src/lib/SwapView.svelte "https://raw.githubusercontent.com/trilitech/tutorial-applications/main/liquidity-baking-dapp/src/lib/SwapView.svelte"
```

File: src/pages/tutorials/build-your-first-app/setting-up-app.md (hunk 1 above)
Comment (beatalipska) [reply src/pages/tutorials/build-your-first-app/setting-up-app.md:649]: Fair enough - but would be good to provide URL for each raw file next to it eg.: `src/config.ts` -> `https://raw.githubusercontent.com/trilitech/tutorial-applications/main/liquidity-baking-dapp/src/config.ts`

**Review by beatalipska — APPROVED** [review]
I had only small suggestions, but no deal-breakers, well done, Tim! 👏🏻 The tutorial is way more readable and easier to go through.

**Review by claudebarde — APPROVED** [review]

**Pull request merged**